proto/executor/v1/executor.proto

```diff
@@ -29,6 +29,11 @@ message ProcessBatchRequest {
     // For testing purposes only
     map<string, string> db = 15;
     map<string, string> contracts_bytecode = 16; // For debug/testing purpposes only. Don't fill this on production
+    // flags for trace debug
+    uint32 disable_storage = 17;
```

Comment:
This new fields should go inside a new message type called TraceConfig and not be directly added to the ProcessBatchRequest

Reply:
In this case, should we add the params `tx_hash_to_generate_execute_trace` and `tx_hash_to_generate_call_trace`to that new message?

Reply:
IDK... maybe.

The idea was to follow Geth "standard" but I guess your proposal is a good idea, yes.

https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#traceconfig